src/ansible_navigator/actions/quit.py

```diff
@@ -21,12 +21,10 @@ def __init__(self, args: ApplicationConfiguration):
 
     # pylint: disable=unused-argument
     def run(self, interaction: Interaction, app: AppPublic) -> Interaction:
-        """Handle :doc
+        """Handle a request to quit the application.
 
         :param interaction: The interaction from the user
-        :type interaction: Interaction
         :param app: The app instance
-        :type app: App
         """
         # : quit will be handled in the app
```

Comment:
How does this comment align with the docstring? Not sure I understand the message it tries to convey.

Reply:
I looked right past it, there's some history there because of how `:quit` used to be handled, let me fix it